values/common-values.yaml

```
@@ -1,7 +1,18 @@
 unregisterRunners: true
 
+
+# Create a non-default service account with the name gitlab-runner with minimal permissions
 rbac:
-  create: false
+  create: true
+  generatedServiceAccountName: gitlab-runner
+  rules:
+    - resources: [""]
+    - apiGroups: [""]
+      resources: [""]
+      verbs: ["*"]
+    - apiGroups: [""]
+      resources: [""]
+      verbs: [""]
```

```suggestion
    - apiGroups: [""]
      resources: [""]
      verbs: [""]
```
Wouldn't this be good enough?